ArduPlane/Attitude.cpp

```diff
@@ -12,14 +12,14 @@ float Plane::calc_speed_scaler(void)
         if (aspeed > auto_state.highest_airspeed && hal.util->get_soft_armed()) {
             auto_state.highest_airspeed = aspeed;
         }
+        // ensure we have scaling over the full configured airspeed
+        const float scale_min = MIN(0.5, g.scaling_speed / (2.0 * aparm.airspeed_max));
+        const float scale_max = MAX(2.0, g.scaling_speed / (0.7 * aparm.airspeed_min));
```

Comment:
This would ensure we don't see low speed oscillations thanks to this fix.
```suggestion
        const float scale_max = 2.0;
```

Reply:
i think we do need to honor the users airspeed min